purchase_discount/views/purchase_discount_view.xml

```diff
@@ -1,39 +1,40 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-  <record model="ir.ui.view" id="purchase_order_line_form2">
-    <field name="name">purchase_discount.order.line.form2</field>
-    <field name="model">purchase.order.line</field>
-    <field name="inherit_id" ref="purchase.purchase_order_line_form2"/>
-    <field name="arch" type="xml">
-      <field name="price_unit" position="after">
-        <field name="discount"/>
-      </field>
-    </field>
-  </record>
 
-  <record model="ir.ui.view" id="purchase_order_line_tree">
-    <field name="name">purchase_discount.order.line.tree</field>
-    <field name="model">purchase.order.line</field>
-    <field name="inherit_id" ref="purchase.purchase_order_line_tree"/>
-    <field name="arch" type="xml">
-      <field name="price_unit" position="after">
-        <field name="discount"/>
-      </field>
-    </field>
-  </record>
+    <record model="ir.ui.view" id="purchase_order_line_form2">
```

Comment:
Please don't change indentation for not adding extra noise, although this doesn't conform 4 spaces indentation.

Reply:
This is still pending